src/features/rewards/tablePending/style.ts

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const StyledRemove = styled<{}, 'button'>('button')`
+  color: #9E9FAB;
+  padding: 0;
+  border: none;
+  background: none;
+  cursor: pointer;
+  text-align: center;
+  width: 100%;
+`
+
+export const StyledRemoveIcon = styled<{}, 'span'>('span')`
+  vertical-align: middle;
+  color: #9E9FAB;
```

Comment:
Do we have a value from the palette we can grab for this one? (Ditto for the one in `StyledRemove`)

Reply:
this color is not in the panel, we could find similar one

Reply:
`#A0A1B2` (grey400) seems to look pretty close

Current:
<img width="85" alt="screen shot 2019-02-05 at 12 17 37 am" src="https://user-images.githubusercontent.com/8732757/52258230-8375e000-28db-11e9-9f37-c89ca7a861db.png">

grey400:
<img width="81" alt="screen shot 2019-02-05 at 12 17 50 am" src="https://user-images.githubusercontent.com/8732757/52258241-8cff4800-28db-11e9-997c-66d27623c2f4.png">

Reply:
done